### PR TITLE
Domain sample cleanup

### PR DESCRIPTION
**Pull request by doxiao**
1) Use domainPVMountPath for the default logHome
2) Add secret name option to create credentials script and run more testing
3) Update the README.md
4) Fix the default pvc

### CODE REVIEW

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 ## Prerequisites
 
 The following prerequisites must be handled prior to running the create domain script:
 * Make sure the WebLogic Operator is running.
-* Create a Kubernetes namespace for the domain custom resource unless the intention is to use the default namespace.
-* Create in the same Kubernetes namespace, the Kubernetes persistent volume where the domain home will be hosted, and the Kubernetes persistent volume claim for the domain resource. For samples to create a PV and PVC, refer to [Create sample PV and PVC](../../create-weblogic-domain-pv-pvc/README.md).
-* Create the Kubernetes secrets `username` and `password` of the admin account in the same Kubernetes namespace as the domain custom resource.
+* Create a Kubernetes namespace for the domain unless the intention is to use the default namespace.
+* Create in the same Kubernetes namespace, the Kubernetes persistent volume where the domain home will be hosted, and the Kubernetes persistent volume claim for the domain. For samples to create a PV and PVC, refer to [Create sample PV and PVC](../../create-weblogic-domain-pv-pvc/README.md).
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md]: Create in the same Kubernetes namespace, the Kubernetes persistent volume -> In the same Kubernetes namespace, create the Kubernetes persistent volume where the ...

File: kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh [hunk 2]
```diff
@@ -51,13 +64,22 @@ while getopts "hu:p:n:d:" opt; do
...
     ;;
     *) usage 1
     ;;
   esac
 done
-secretName=$domainUID-weblogic-credentials
+
+if [ -z $secretName ]; then
+  if [ -z $domainUID ]; then
+    secretName=weblogic-credentials
```
Comment (alai8) [inline kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: The comment earlier says the default secretName would be domain1-weblogic-credentials when neither secretName or domainUID are provided

Comment (doxiao) [reply kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: The script initializes the domainUID to domain1. So if the domainUID is not set here, which means that it is overridden by the -d option with "". So we now fall into case #4.

Comment (doxiao) [reply kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: hmmm, maybe I need to check if the domainUID is not to ""? I tested all permutations. Let me test again.

Comment (doxiao) [reply kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: case #1: 
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain40 -s mysecret
secret "mysecret" created
secret "mysecret" labeled
The secret mysecret has been successfully created in the default namespace.

case #2:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1
secret "domain1-weblogic-credentials" created
secret "domain1-weblogic-credentials" labeled
The secret domain1-weblogic-credentials has been successfully created in the default namespace.

Case #3:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain40
secret "domain40-weblogic-credentials" created
secret "domain40-weblogic-credentials" labeled

case #4:
bash-4.2$ sh create-weblogic-credentials.sh -u weblogic -p welcome1 -d ""
secret "weblogic-credentials" created
The secret weblogic-credentials has been successfully created in the default namespace.

Comment (alai8) [reply kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: Thanks for clarifying! 

Do we support "" as valid domainUID? 

Do you think the comment for #4 should be updated to 
4) weblogic-credentials - if secretName is not specified, and domainUID is specified as "".

Comment (doxiao) [reply kubernetes/samples/scripts/create-weblogic-domain/create-weblogic-credentials.sh]: Good catch on a copy-and-paste error.

**Pull request merged**